xml/System/Char.xml

```diff
@@ -3142,7 +3150,7 @@ The following code example demonstrates <xref:System.Char.IsLetter%2A>.
 
 -   Currency symbols.
 
--   Letterlike symbols, which include a set of mathematical alphanumeric symbols as well as symbols such as ℅, №, and ™.
+-   Letterlike symbols, which include a set of mathematical alphanumeric symbols as well as symbols such as ?, ?, and �.
```

Comment:
The data got lost here. I think the tool had trouble with the encoding. Can you please fix it, @danmoseley ?

Same request in the other lines below.